app/services/physical_infra_topology_service.rb

```diff
@@ -67,8 +67,12 @@ def entity_status(entity)
     case entity
     when ManageIQ::Providers::PhysicalInfraManager
       entity.authentications.blank? ? _('Unknown') : entity.authentications.first.status.try(:capitalize)
-    when PhysicalServer, PhysicalSwitch
+    when PhysicalServer, PhysicalSwitch, PhysicalStorage, PhysicalChassis
       entity.health_state ? entity.health_state : _('Unknown')
+    when ManageIQ::Providers::Redhat::InfraManager::Host
+      entity.state ? entity.state.downcase.capitalize : _('Unknown')
+    when ManageIQ::Providers::Redhat::InfraManager::Vm
+      entity.power_state ? entity.power_state.downcase.capitalize : _('Unknown')
```

Comment:
Why just `Redhat::InfraManager`, what about all the others? Shouldn't be these simply just `Host` and `Vm`?

Reply:
You're right. I fixed these unnecessary namespaces use. :smile: